### PR TITLE
Updated TensorFlow logo

### PR DESCRIPTION
**Pull request by kyscg**
I think this is a necessary change @rthadur 

The repo still shows the old logo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div align="center">
-  <img src="https://www.tensorflow.org/images/tf_logo_transp.png"><br><br>
+  <img src="https://www.gstatic.com/devrel-devsite/vd93acc801b2ca40996c3d3af42fda68a31a452afd3dccab01446b6f5e8d85a31/tensorflow/images/lockup.svg"><br><br>
```
Comment (arp95) [inline README.md]: Is this the official site of the logo? I think having an official url would be beneficial.

Comment (kyscg) [reply README.md]: Yes, I've taken it by inspecting element (logo) from tensorflow.org

**Comment by drpngx** [comment]
That doesn't seem right. I think the [image](https://www.tensorflow.org/images/tf_logo_transp.png) has been updated.  You might have caught this before we had a chance to update. Closing.

**Comment by kyscg** [comment]
But it doesn't show the changes on the repo, and my PR fixes that

**Pull request closed without merging**